Library/Formula/rethinkdb.rb

```diff
@@ -10,11 +10,9 @@ class Rethinkdb < Formula
   end
 
   depends_on :macos => :lion
-  # Embeds an older V8, whose gyp still requires the full Xcode
-  # Reported upstream: https://github.com/rethinkdb/rethinkdb/issues/2581
-  depends_on :xcode => :build
   depends_on "boost" => :build
   depends_on "openssl"
+  depends_on "icu4c"
```

Comment:
OS X does actually ship `icu4c` but just with missing headers etc. I wonder if this can be worked around somehow?